pkg/cmd/spec/generator.go

```diff
@@ -9,7 +9,8 @@ import (
 
 	"github.com/acarl005/stripansi"
 	"github.com/pterm/pterm"
-	"gopkg.in/yaml.v3"
+	"gopkg.in/yaml.v2"
+	yamlv3 "gopkg.in/yaml.v3"
```

Comment:
Can you explain the differences between the yamlv2 and yamlv3 modules that you have imported?

Reply:
Sure, yaml.v2 has the type of `MapSlice` which we are using to maintain the order of container's environment variables. However, yaml.v3 doesn't has this type, and thus we should import both of the packages here.

Reply:
Sure, yaml.v2 has the type of `MapSlice` which we are using to maintain the order of container's environment variables. However, yaml.v3 doesn't has this type, and thus we should import both of the packages here.

Reply:
If yamlv2 is necessary, please add some comments about it. Can we change yamlv3 to yamlv2 in this file?

Reply:
OK, I've added some comments. It's sad that we can't just replace yaml.v3 with yaml.v2 in this file because the method `KnownFields` called in `GenerateSpecFromFile` is undefined in yaml.v2.